src/settings/index.ts

```diff
@@ -94,12 +94,12 @@ export const IFRAME_TEMPLATES = [
   VenueTemplate.performancevenue,
   VenueTemplate.posterpage,
   VenueTemplate.viewingwindow,
-];
+]);
 
-export const EMBEDDABLE_CONTENT_TEMPLATES = [
+export const EMBEDDABLE_CONTENT_TEMPLATES = Object.freeze([
   ...IFRAME_TEMPLATES,
   ...ZOOM_URL_TEMPLATES,
-];
+]);
```

Comment:
[NOTE] This is nor request to change, just an FYI.

In most cases I use this form:

```ts
const object = {};
Object.freeze(object);
```

just because in the first case TS might infer the type as `Readonly<Record<T,R>>`, while in the snippet I provided is `Record<T,R>`. Usually I'd prefer the way you wrote it, but in some cases, it might be easier than dealing with type complaints.